### PR TITLE
Fix Typo

### PR DESCRIPTION
**Pull request by akrishna0**
In .ansible-lint.yml fixed a typo of that to then and added a comma between push and add.

### CODE REVIEW

File: docs/disabling-linters.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Disabling linters and Rules
 If you find you need to ignore certain **errors** and **warnings**, you will need to know the *format* to disable the **Super-Linter** rules.  
 Below is examples and documentation for each language and the various methods to disable.
```
Comment (matmunn) [inline docs/disabling-linters.md:3]: This line seems to be duplicated instead of replaced

File: docs/disabling-linters.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-
+Below are examples and documentation for each language and  various methods to disable.
```
Comment (zkoppert) [inline docs/disabling-linters.md:4]:
```suggestion
Below are examples and documentation for each language and various methods to disable.
```

File: docs/disabling-linters.md (hunk 1 above)
Comment (akrishna0) [reply docs/disabling-linters.md:3]: Okay I will make the changes.

File: docs/disabling-linters.md (hunk 2 above)
Comment (akrishna0) [reply docs/disabling-linters.md]: I have made the changes please review

**Comment by akrishna0** [comment]
> Hi @akrishna0! Thanks for submitting this! tada Can you fix the issue pointed out by @matmunn and review the removal of the extra space I suggested?

I have made the suggested changes please review.

**Pull request merged**